domains/auth/Auth.API/Tests/TestLogout.cs

```diff
@@ -0,0 +1,61 @@
+using API.Services;
+using API.Configuration;
+using Microsoft.Extensions.Options;
+using Xunit;
+using Xunit.Categories;
+using Moq;
+using API.Controllers;
+using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using System.Linq;
+
+namespace Tests;
```

Comment:
Should we arrange this in a Test.Service dir so we can get some structure in our test projects.?

Reply:
Should'nt be Test.Controller then?